### DISCUSSION TITLE
WebStorm Copilot version 1.1.34.2009 is terrible!

### DISCUSSION THREAD

**Discussion by MichaelLeeHobbs** [community/community#37043] [Copilot Conversations] [answered]
I have not updated for few weeks so not sure what the pervious version was I had installed. Today after updating, Copilot rarely has any suggestions. The new side panel suggestions are terrible not even remotely close to useful. It's more like it pulled random snippets from other projects I had open. After rolling back to 1.1.33.1980 it seems to be better. The side panel had one 94 line suggestion for a file I was refactoring and it was exactly what I wanted. Will add an update if I find 1.1.33.1980 is not performing as expected based on previous experiences with Copilot.

**Comment by iamcjwilliams** [comment, marked as answer]
@MichaelLeeHobbs Thank you for your feedback, if you're still having issues with version 1.1.34.2009, can you provide a screenshot by performing (View -> Output; select GitHub Copilot from the pulldown)?

**Reply by MichaelLeeHobbs** [reply]
No issues with 1.1.35.2063
